ui/app/models/job.js

```diff
@@ -167,8 +168,17 @@ export default class Job extends Model {
     }, []);
   }
 
-  runAction(action, allocID) {
-    return this.store.adapterFor('job').runAction(this, action, allocID);
+  /**
+   *
+   * @param {ActionModel} action
+   * @param {string} allocID
+   * @param {import('../models/action-instance').default} actionInstance
```

Comment:
TODO: I'm importing model definitions in two different ways here for no real reason.